src/__tests__/Nav.spec.js

```diff
@@ -1,94 +1,83 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { screen, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import { Nav } from '..';
 
 describe('Nav', () => {
   it('should render .nav markup', () => {
-    const wrapper = shallow(<Nav />);
-
-    expect(wrapper.html()).toBe('<ul class="nav"></ul>');
+    let { container } = render(<Nav />);
+    expect(container).toContainHTML('<ul class="nav"></ul>');
   });
 
   it('should render custom tag', () => {
-    const wrapper = shallow(<Nav tag="nav" />);
-
-    expect(wrapper.html()).toBe('<nav class="nav"></nav>');
+    let { container } = render(<Nav tag="nav" />);
```

Comment:
Can we use `testForCustomTag` from `testUtils.js` here? That should help with consistency.